src/backend/src/services/reimbursement-requests.services.ts

```diff
@@ -89,7 +97,7 @@ export default class ReimbursementRequestService {
 
     if (!expenseType) throw new NotFoundException('Expense Type', expenseTypeId);
```

Comment:
can you enforce that a reimbursement request can't be created unless a user has their secure settings set in the backend/frontend